packages/phrases/src/locales/en/translation/admin-console/applications.ts

```diff
@@ -1,7 +1,7 @@
 const applications = {
   title: 'Applications',
   subtitle:
-    'Set up a mobile, single page or traditional application to use Logto for authentication',
+    'Set up Logto authentication for your native, single page, M2M, or traditional application',
```

Comment:
machine-to-machine?

Reply:
Remenber to add the `// UNTRANSLATED` tag to other locale's related translation.
You can resolve this comment by yourself once done.

Reply:
Let's use `machine to machine` and use it in all languages, no neeed to add `// UNTRANSLATED`. Will update soon